cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -95,6 +100,18 @@ function ManagementCredentials(props: Partial<StepProps>) {
             if (goToStep && currentStep && submitForm) {
                 goToStep(currentStep + 1);
                 submitForm(currentStep);
+                awsDispatch({
+                    type: AWS_ADD_RESOURCES,
+                    resourceName: 'osImages',
+                    payload: osImages,
+                } as AwsResourceAction);
+                if (osImages[0]) {
```

Comment:
I'm wondering if we should be checking local storage to see if the user has previously selected an image, and if that image is in our osImages list, should we use it?

Reply:
Yes sir. That makes sense. Gonna add logic here

Reply:
For 'local storage', do you mean the browser local storage? We don't use local storage currently, right?

Reply:
Correct that we are not using (browser) local storage at this time, so I'm marking this conversation as resolved.

Reply:
Got it. Maybe we can use that in the future. I am gonna do some research about that